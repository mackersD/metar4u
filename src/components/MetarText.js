import React from 'react'

const MetarText = ({text}) => {
  return (
    <div>
      {text}
    </div>
  )
}

export default MetarText
