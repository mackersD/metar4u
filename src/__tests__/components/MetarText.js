import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import MetarText from '../../components/MetarText'

it('MetarText - renders', () => {
  const div = document.createElement('div')
  ReactDOM.render(<MetarText />, div)
});

it('MetarText - expected rendering', () => {
  var comp = renderer.create(<MetarText text={"KGTU"} />).toJSON()
  expect(comp).toMatchSnapshot()
})
