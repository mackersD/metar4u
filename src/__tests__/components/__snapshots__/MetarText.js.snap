// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MetarText - expected rendering 1`] = `
<div>
  KGTU
</div>
`;
